### PR TITLE
[Enhancement] - Change Settings > Help web views to Chrome Custom Tabs

### PR DESCRIPTION
**Pull request by Rcureton**
# What ❓
- Changed all of the Help links that were previously using `WebViews` and refactored them to using `ChromeTabs` which give these links a more native feel and faster loading.
- Unfortunately Chrome Tabs doesn't support injecting headers as per this article [Chrome tabs  and headers](https://medium.com/@_rpiel/chrome-custom-tabs-and-headers-or-a-happy-open-source-story-9c3f82680eea). Also I referenced the documentation on Chrome tabs and there's nothing for injecting headers as well.

- I used this article as a reference for implementation [Chrome Custom Tabs](https://developer.chrome.com/multidevice/android/customtabs#bestpractices)

# Story 📖
[Change Settings > Help web views to Chrome Custom Tabs]( https://trello.com/c/RNha843j/1152-change-settings-help-web-views-to-chrome-custom-tabs)

# See 👀
feb-01-2019 16-14-06

### CODE REVIEW

**Comment by Rcureton** [comment]
[Change Settings > Help web views to Chrome Custom Tabs](https://trello.com/c/RNha843j/1152-change-settings-help-web-views-to-chrome-custom-tabs)

File: app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt [hunk 1]
```diff
@@ -85,4 +74,18 @@ class HelpSettingsActivity : BaseActivity<HelpSettingsViewModel.ViewModel>() {
             startActivity(Intent.createChooser(intent, getString(R.string.support_email_chooser)))
         }
     }
+
+    private fun loadChromePages(endpoint: String) {
```
Comment (eoji) [inline app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt]: Let's rename this to either `startChromeTab` or `startCustomTab` for accuracy.

File: app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt [hunk 2]
```diff
@@ -85,4 +74,18 @@ class HelpSettingsActivity : BaseActivity<HelpSettingsViewModel.ViewModel>() {
...
+        webEndpointBuilder.appendEncodedPath(endpoint)
+
+        val url = webEndpointBuilder.build().toString()
+        val builder = CustomTabsIntent.Builder()
+
+        builder.setShowTitle(true)
+        builder.setToolbarColor(ContextCompat.getColor(this, R.color.primary))
+
+        val customTabsIntent = builder.build()
+        customTabsIntent.launchUrl(this, Uri.parse(url))
```
Comment (eoji) [inline app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt]: This crashes if the user doesn't have a browser that's capable of handling a custom tabs.  I don't think we needed to remove all of that code. We need a fallback.

File: app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt [hunk 3]
```diff
@@ -46,21 +46,10 @@ class HelpSettingsActivity : BaseActivity<HelpSettingsViewModel.ViewModel>() {
...
-        }
-
-        privacy_policy.setOnClickListener {
-            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.Privacy::class.java))
-        }
-
-        terms_of_use.setOnClickListener {
-            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.Terms::class.java))
-        }
+        cookie_policy.setOnClickListener{ loadChromePages("cookies") }
```
Comment (eoji) [inline app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt]: These should be constants. We previously used `Secrets.HelpCenter.ENDPOINT` and the static classes in `HelpActivity`

**Comment by Rcureton** [comment]
Done!

**Pull request merged**